tests/bluetooth/tester/src/gap.c

```diff
@@ -159,6 +159,7 @@ static void le_security_changed(struct bt_conn *conn, bt_security_t level,
 	const bt_addr_le_t *addr = bt_conn_get_dst(conn);
 	struct gap_sec_level_changed_ev sec_ev;
 	struct gap_bond_lost_ev bond_ev;
+	struct bt_conn_info info;
```

Comment:
This is only used within the `case BT_SECURITY_ERR_PIN_OR_KEY_MISSING:` Consider moving the declaration to there.

Reply:
this will not compile (unless extra { } are used)

Reply:
I know :) 

It's fine to keep as is